ts/Extensions/DashboardPlugins/HighchartsComponent.ts

```diff
@@ -301,9 +309,21 @@ class HighchartsComponent extends Component<HighchartsComponent.ChartComponentEv
 
         table.setCell(columnName, rowNumber, valueToSet);
     }
+    /**
+     * Handles updating via options
+     * @param {Partial<Component.ComponentOptions>} options
+     * The options to apply
+     *
+     * @param {boolean} redraw
+     * The flag triggers the main redraw operation
+     */
+    private updateComponentOptions(options: Partial<HighchartsComponent.ComponentOptions>, redraw = true): void {
```

Comment:
Shouldn't eslint complain here about the line length?

Reply:
Because of generic typing it will not complaint.

Reply:
No errors here ;)

Reply:
Can you please fix it and split over multiple lines. The eslint ignore is just for overlengthy generic types, which is not the case here.